### PR TITLE
Specify unnamed libraries

### PR DESCRIPTION
**Pull request by srawlins**

### CODE REVIEW

**Review by leafpetersen — APPROVED** [review]
LGTM, thanks!

File: accepted/future-releases/unnamed-libraries/feature-specification.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+### Parts
+
+A library part specifies the library to which it belongs using the part-of
+directive, which accepts two ways of refering to a library. A part-of directive
+can specify a library by URI, which is the more common way, and does not
+require the library to be explicitly named. In an older style, a part-of
+directive can instead specify a library by its name. A part-of directive cannot
+refer by name to an implicitly named library.  Therefore, with this feature, a
+part-of directive cannot refer to a library with a library directive without a
```
Comment (lrhn) [inline accepted/future-releases/unnamed-libraries/feature-specification.md:65]: ... a part-of directive using a library name cannot ...

**Review by lrhn — APPROVED** [review]

File: accepted/future-releases/unnamed-libraries/feature-specification.md (hunk 1 above)
Comment (srawlins) [reply accepted/future-releases/unnamed-libraries/feature-specification.md:65]: Thanks!

File: accepted/future-releases/unnamed-libraries/feature-specification.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+becomes:
+
+> ```latex
+> <libraryName> ::= <metadata> \LIBRARY{} <dottedIdentifierList>? `;'
+> ```
+
+### Parts
+
+A library part specifies the library to which it belongs using the part-of
+directive, which accepts two ways of refering to a library. A part-of directive
```
[the remark this answers is not in the document]
Comment (srawlins) [reply accepted/future-releases/unnamed-libraries/feature-specification.md:60]: Thanks!

**Pull request merged**